packages/ui/core-components/src/lib/unsorted/viz/core/_Value.svelte

```diff
@@ -101,9 +104,17 @@
 		>[{placeholder}]<span class="error-msg">Placeholder: no data currently referenced.</span></span
 	>
 {:else if !error}
-	<span>
-		{formatValue(selected_value, format_object)}
-	</span>
+	{#if link}
+		<a class="hover:bg-blue-100" href={link}>
```

Comment:
not sure about this background. is it obvious enough?

should we just do class="markdown" so it's styled like a link?

Reply:
not familiar but ill take a look with the markdown class